llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -1681,13 +1682,10 @@ static Expected<CUInfo> getAndSetDebugAbbrevOffsetAndSkip(
       return std::move(E);
   }
 
-  // Do not skip the CU length when materializing the debug info section, the CU
-  // length will be larger than the CUData length.
-  if (SkipData) {
     Reader.setOffset(AfterSizeOffset);
     if (auto E = Reader.skip(*Size))
       return std::move(E);
```

Comment:
Shouldn't the indentation change here? Or is this Github trolling us?

Reply:
Ah maybe I forgot to do a `git clang-format` here, I will fix it, sorry!